src/@context/Asset.tsx

```diff
@@ -77,7 +77,7 @@ function AssetProvider({
         return
       }
 
-      if (asset.nft.state) {
+      if ([1, 2, 3].includes(asset.nft.state)) {
         // handle nft states as documented in https://docs.oceanprotocol.com/concepts/did-ddo/#state
```

Comment:
while we are at it, this URL does not exist. It's `https://docs.oceanprotocol.com/core-concepts/did-ddo#state`

Reply:
that was an old comment,  good catch